tests/properties/pflua_pipelines_match.lua

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env luajit
+-- -*- lua -*-
+module(..., package.seeall)
+package.path = package.path .. ";../?.lua;../../src/?.lua"
+-- Compare the results of the libpcap/bpf and pure-lua pflua pipelines.
+
+local pf = require("pf")
+local savefile = require("pf.savefile")
+local utils = require('pf.utils')
+
+local pflang = require('pfquickcheck.pflang')
+
+function property(packets)
+   --nil pkt_idx, pflang_expr, bpf_result, pflua_result to avoid
+   -- confusing debug information
+   pkt_idx, pflang_expr, bpf_result, pflua_result = nil
+   local pkt, P, pkt_len, libpcap_pred, pflua_pred
+   a = pflang.Pflang()
+   pflang_expr = table.concat(a, ' ')
```

Comment:
Similar comment to about arithmetic expressions, above - at the moment, I'm generating a limited enough subset of pflang that I believe this isn't an issue, because that limited subset has been enough to find a lot of bugs and makes for beautifully simple test cases and bug reports (and I believe it's never invalid, including without parens). If/when it makes sense to generate richer data, you're absolutely right.